pkg/api/secretbootstrap/secretboostrap.go

```diff
@@ -45,7 +47,7 @@ type DockerAuth struct {
 
 type SecretContext struct {
 	// A cluster to target. Mutually exclusive with 'ClusterGroups'
-	Cluster string `json:"cluster"`
+	Cluster string `json:"cluster,omitempty"`
```

Comment:
This is correct since a group may be set? Do we have validation to ensure that `cluster = ""` ensures a group is set?

Reply:
I don't believe so, I will research and add that if needed.

Reply:
This validation also currently exists in `cmd/ci-secret-bootstrap/main.go`.